basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/clustering/BroadcastGroupsTestCase.java

```diff
@@ -68,7 +68,6 @@ public static void afterClass() throws IOException, InterruptedException, Operat
         operations.removeIfExists(BG_TBR_ADDRESS);
         administration.reloadIfRequired();
         operations.removeIfExists(NEW_SERVER);
-        administration.reloadIfRequired();
```

Comment:
Reload is unnecessary since it's performed again in parent `AfterClass` method. Order of `AfterClass` invocations is guaranteed by JUnit Javadoc. 
Another question is if using testcase class inheritance to reuse part of setup and cleanup code is good idea ... probably not (bad readability, error prone) and should be refactored!